src/Tool/TokenStorageActorProvider.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Stof\DoctrineExtensionsBundle\Tool;
+
+use Gedmo\Tool\ActorProviderInterface;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * Provides an actor for the extensions using the token storage.
+ */
+class TokenStorageActorProvider implements ActorProviderInterface
```

Comment:
Can you make the class final and tag it as `@internal` ? I don't see any reason why a project would need to extend that class (they can implement the interface if they want) and I would prefer not having to maintain BC only for an hypothetical use case.

Reply:
Done!